src/App.tsx

```diff
@@ -17,18 +60,39 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                inputValue={inputValue}
+                setInputValue={setInputValue}
+                selectOption={selectOption}
+                setSelectOption={setSelectOption}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length === 0
+                ? <Loader />
```

Comment:
It's better practice to create a separate state to track wheter the data isLoading and show spinner only if that state is true. Because the request may return with empty data, and that case should be handled differently